library/src/test/java/com/datatorrent/lib/appdata/datastructs/DimensionalTableTest.java

```diff
@@ -38,7 +38,7 @@ public void serializabilityTest() throws Exception
   {
     DimensionalTable<Integer> table = createTestTable();
 
-    TestUtils.clone(new Kryo(), table);
+    KryoCloneUtils.cloneByKryo(new Kryo(), table);
```

Comment:
This method is still necessary, because people may want to use their own customized Kryo